circuit/js/test/unit/encoder.test.ts

```diff
@@ -19,8 +19,9 @@ describe("Encoder", () => {
       maxOutputs: AxiomV2CircuitConstant.UserMaxOutputs,
       isAggregation: false,
     }
+    console.log("DEFAULT_METADATA", DEFAULT_METADATA);
     const encoded = encodeAxiomV2CircuitMetadata(DEFAULT_METADATA);
-    const expected = "0x0001000009000100000004010000010080000000000000000000000000000000";
+    const expected = "0x0001000009000100000014030000010080000000000000000000000000000000";
```

Comment:
needed to modify this based on the new wider circuit config in tools: https://github.com/axiom-crypto/axiom-tools/commit/d25d64ac9cec57257ec7ebe513828f7e89a6ffb4

Reply:
This matches https://github.com/axiom-crypto/axiom-eth-working/pull/238